examples/fleetpub.ts

```diff
@@ -0,0 +1,52 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import RclnodejsTransport from '@osrf/romi-js-rclnodejs-transport';
+
+//example of how one could publish fleet state to the  web
```

Comment:
technically you are publishing to the rmf system instead of to the web specifically. Any rmf nodes will be able to receive the fleet states, not just romi-js based nodes.